src/main/java/us/kilroyrobotics/subsystems/Wrist.java

```diff
@@ -27,11 +115,28 @@ public Pose3d getWristPose() {
                 0.0254,
                 this.getCarriagePose.get().getZ() + 0.2899918,
                 new Rotation3d(
-                        Degrees.of(0), Degrees.of(0) /* Wrist angle goes here */, Degrees.of(0)));
+                        Degrees.of(0),
+                        Radians.of(
+                                (this.m_useAbsoluteEncoder
```

Comment:
See above comment about relative encoder

Reply:
Response in comment